frontend/src/module.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module '@carbon/icons-vue/es/checkmark--filled/20'
+declare module '@carbon/icons-vue/es/close/16';
+declare module '@carbon/icons-vue/es/user--follow/16';
+declare module '@carbon/icons-vue/es/help/16'
+declare module '@carbon/icons-vue/es/error--filled/32';
+declare module '@carbon/icons-vue/es/add/20';
+declare module '@carbon/icons-vue/es/search/16';
+declare module '@carbon/icons-vue/es/user--avatar--filled/20';
+declare module '@carbon/icons-vue/es/user--avatar/20';
+declare module '@carbon/icons-vue/es/login/20';
```

Comment:
Wondering, why do we need this d.ts?

Reply:
Try `npm i --save-dev @types/carbon__icons-vue` if it exists or add a new declaration (.d.ts) file containing `declare module '@carbon/icons-vue/es/help/16'
Typescript was complaining about the types and recommended adding a new declaration (.d.ts) file containing `declare module`

Reply:
I just removed the module.d.ts since all the icons are now using the icon component

Reply:
That's great! Thanks Jota.